.bumpversion.cfg

```diff
@@ -1,8 +0,0 @@
-[bumpversion]
-current_version = 1.7.0
-commit = True
-tag = False
-
-[bumpversion:file:publicCloudInfo-server.spec]
-search = Version:   {current_version}
-replace = Version:   {new_version}
```

Comment:
I see no reason to remove the bumpversion setup.

Reply:
New version would be 2.0.0

Reply:
This file is for the Ruby package. For the Python package, we are using PBR to manage versioning.